admin/templates/settings/standard-source.php

```diff
@@ -2,35 +2,43 @@
 /**
  * Render the Exclude setting for the "Use standard source" option
  *
- * @var string $standard_source value of the Standard Source option
+ * @var string $standard_source      value of the Standard Source option
  * @var string $standard_source_text text in the Standard Source Text option
  */
 ?>
 <p><?php esc_html_e( 'Choose how to handle images with the “Use standard source” option enabled.', 'image-source-control-isc' ); ?></p>
 <br/>
 <div class="isc-settings-highlighted isc-settings-standard-source">
-<label>
-	<input type="radio" name="isc_options[standard_source]" value="exclude" <?php checked( $standard_source, 'exclude' ); ?> />
-	<?php esc_html_e( 'Exclude from lists', 'image-source-control-isc' ); ?>
-</label>
-<p class="description"><?php esc_html_e( 'Hide images sources in the frontend. You can still manage them in the dashboard.', 'image-source-control-isc' ); ?></p>
-<label>
-	<input type="radio" name="isc_options[standard_source]" value="author_name" <?php checked( $standard_source, 'author_name' ); ?> />
-	<?php esc_html_e( 'Author name', 'image-source-control-isc' ); ?>
-</label>
-<p class="description"><?php esc_html_e( 'Use the uploader’s public name as the image source.', 'image-source-control-isc' ); ?></p>
-<label>
-	<input type="radio" name="isc_options[standard_source]" id="isc-custom-text-select" value="custom_text" <?php checked( $standard_source, 'custom_text' ); ?> />
-	<?php esc_html_e( 'Custom text', 'image-source-control-isc' ); ?>
-</label>
-<input type="text" id="isc-custom-text" name="isc_options[standard_source_text]" value="<?php echo esc_attr( $standard_source_text ); ?>" <?php disabled( $standard_source != 'custom_text' ); ?> class="regular-text" placeholder="<?php echo esc_attr( $this->get_standard_source_text() ); ?>"/>
+	<label>
+		<input type="radio" name="isc_options[standard_source]" value="exclude" <?php checked( $standard_source, 'exclude' ); ?> />
+		<?php esc_html_e( 'Exclude from lists', 'image-source-control-isc' ); ?>
+	</label>
+	<p class="description"><?php esc_html_e( 'Hide image sources in the frontend.', 'image-source-control-isc' ); ?></p>
+	<label>
+		<input type="radio" name="isc_options[standard_source]" value="author_name" <?php checked( $standard_source, 'author_name' ); ?> />
+		<?php esc_html_e( 'Author name', 'image-source-control-isc' ); ?>
+	</label>
+	<p class="description"><?php esc_html_e( 'Use the uploader’s public name as the image source.', 'image-source-control-isc' ); ?></p>
+	<label>
+		<input type="radio" name="isc_options[standard_source]" id="isc-custom-text-select" value="custom_text" <?php checked( $standard_source, 'custom_text' ); ?> />
```

Comment:
**Suggestion:** Consider using a constant for the string 'custom_text' as it is used multiple times in the code. This will make the code more maintainable and less prone to errors. [maintainability]
```suggestion
  define('DEFAULT_STANDARD_SOURCE', 'custom_text');
  <input type="radio" name="isc_options[standard_source]" id="isc-custom-text-select" value="<?php echo DEFAULT_STANDARD_SOURCE; ?>" <?php checked( $standard_source, DEFAULT_STANDARD_SOURCE ); ?> />
```